riscv/src/riscv_asm.lalrpop

```diff
@@ -109,6 +117,7 @@ Register: Register = {
 
 OffsetRegister: Argument = {
     <c:Expression> "(" <r:Register> ")" => Argument::RegOffset(r, c),
+    "(" <r:Register> ")" => Argument::RegOffset(r, Expression::Number(0)),
```

Comment:
There could actually be a distinction. Could we make it `Option<Expression>`?

Reply:
I am not sure what is the best option for this type. I certainly looks like a `RegOffset` with offset zero, because the register will be used as pointer to memory.

But we probably should do here what GNU assembler would do. I am assuming that this syntax just means offset zero, and that is why they felt appropriate to use it with `amoadd`.

Reply:
My point is that the meaning could depend on the instruction.

Reply:
It is now optional.